### ISSUE TITLE
Add ability to abort processing of a .tst file

### ISSUE THREAD

**Issue by fingolfin** [gap-system/gap#1922]
Sometimes, one wants to run a test only conditionally. Say only on 32 bit systems, or only if a certain package is available, or ...

For this, it would be nice if a test could somehow signal that it wants to be skipped. Something like this:
```
# mytest.tst
gap> if some_condition = false then  SKIP_TEST(); fi;
...
```
Then we need to teach `Test` to skip the test; i.e. it would stopand `TestDirectory` would simply proceed to the next file.

One could also show a message indicating the test was skipped, and perhaps also allow an (optional?) message argument to `SKIP_TEST`, like `SKIP_TEST("this test requires package FOO, which is not loaded")`.

**Comment by fingolfin** [comment]
See also PR #1633

**Comment by fingolfin** [comment]
This was resolved by PR #1633
